server/graphql/v2/object/ParseUploadedFileResult.ts

```diff
@@ -0,0 +1,49 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+import { GraphQLDate } from 'graphql-scalars';
+
+import { GraphQLStrictPercentage } from '../scalar/StrictPercentage';
+
+import { GraphQLAmount } from './Amount';
+
+const GraphQLExpenseParsedFileInfo = new GraphQLObjectType({
+  name: 'ExpenseParsedFileInfo',
+  fields: () => ({
```

Comment:
Can we use typescript to enforce that these fields are compatible with the Expense object?